apps/dot-voting/app/components/VotingCard.js

```diff
@@ -96,13 +96,14 @@ const VotingCard = ({ vote, onSelectVote }) => {
         {options.length > 2 && (
           <div css="text-align: center; width: 100%; margin-top: 10px">
             <Badge
-              shape="compact"
-              foreground={`${theme.surfaceOpened}`}
-              background={`${theme.surfaceUnder}`}
+              foreground={`${theme.surfaceContentSecondary}bf`}
+              background={`${theme.border}7f`}
```

Comment:
What do these `bf` and `7f` do? I haven't seen this pattern before.

Reply:
Yeah, this is ugly and I was really unhappy about doing it. Its just adding alpha values to the colours in hex. In the figma, these values use the `opacity` value in the css but that doesn't work in the component as the opacity on the background is multiplied by the opacity on the foreground making the foreground more transparent than it should be. However, now that you point it out, I could take the rgb values from the theme color and then have the alpha as a percentage rather than the hex. e.g. `foreground = 'rgba(${theme.surfaceContentSecondary.r}, ${theme.surfaceContentSecondary.g}, ${theme.surfaceContentSecondary.b}, 0.7);'` But I feel like that may be a worse alternative.

Reply:
We don't have to exactly copy what is done in Figma if it breaks the conventions of the design system. @jayalaves can you please provide recommendations to what the hex values should be so they are in line with the design system?

Reply:
@PeterMPhillips The closest value as far as hex goes without applying opacity for the shadow is #D9D9D9. How does Aragon One style their vote cards on the Voting app?

Reply:
Please hold @PeterMPhillips there is a miscommunication with the problem here :)

Reply:
@PeterMPhillips Simply use 'surfaceUnder' for the background, and 'contentSecondary' for the text. Those should be nearly if not fully accurate with what we have on Figma.

Reply:
Thank you @jayalaves!
![Screenshot from 2019-11-26 10-19-02](https://user-images.githubusercontent.com/19808076/69660987-58a97380-1036-11ea-80c4-009416bfe46c.png)